.travis.yml

```
@@ -31,7 +36,9 @@ install: jabba install $(jabba ls-remote "adopt@~1.$TRAVIS_JDK.0-0" --latest=pat
 stages:
   - validations
   - test
-  - test-sbt
+  - test-sbt-1.3.x
+  - cron-test-sbt-1.3.x
+  - cron-test-sbt-1.4.x
```

```suggestion
  - test-sbt-1.3.x-2.12
  - cron-test-sbt-1.3.x-2.13
  - cron-test-sbt-1.4.x
```

Indicating the difference (scala version) to make it clearer what's the version tested on every PR.

---

Not a blocker. Let's solve this elsewhere.